src/web/routes/entities/types.ts

```diff
@@ -9,9 +9,7 @@ import type {
 	DisassociateEntityUseCaseParams,
 } from '../../../usecases';
 
-export type EntitiesResponseBody =
-	| { readonly design: AtlassianDesign }
-	| string;
+export type EntitiesResponseBody = AtlassianDesign | string;
```

Comment:
**question:** @stannnous , please, could you remove ` | string` from the type declaration? It seems it is a legacy artifact -- the endpoind should always return `AtlassianDesign` in case of success.